components/QuestionRenderer.tsx

```diff
@@ -1,24 +1,36 @@
-import { RadioButtonProps, RadioButtons } from '../RadioButtons'
+import { useTranslations } from 'next-intl'
+
+import { RadioButtonProps, RadioButtons } from './RadioButtons'
 import {
   SatisfactionSlider,
   SatisfactionSliderProps
-} from '../SatisfactionSlider'
-import { TextResponse, TextResponseProps } from '../TextResponse'
+} from './SatisfactionSlider'
+import { TextResponse, TextResponseProps } from './TextResponse'
 
 export type Question = {
-  // TODO: superior type
+  // TODO: superior type. Needs to allow any type of question
   i18n?: Record<string, any>
   title?: string
```

Comment:
Also add `placeholder` to this type.

Reply:
Added in 8dfb1fa94f1dec475902373644d7635b28ef4af5